### DISCUSSION TITLE
How to customize modal's width?

### DISCUSSION THREAD

**Discussion by blockhunts** [wireui/wireui#604] [Q&A] [answered]
It may sounds noob but, I am still noob after all.
But how to customize the white (content) parts of the modal, so that the size will be bigger?

**Comment by blockhunts** [comment, marked as answer]
Silly me, I only need to add
`max-width="6xl"` inside the x-modal

**Reply by swartjie** [reply]
> max-width="6xl"

Where did you put this?
I've tried everything. I can't get my modal size to change?

i've tried using the constants, manually setting the width. I just can't get the width wider...

My example modal:

```
<x-modal name="addUserRoleModal">
        <x-card title="Add User Role">
            <x-input label="Role Name" wire:model="role.name"/>
            <x-input label="Role Description" wire:model="role.description"/>

            <x-slot name="footer" class="flex justify-end gap-x-4">
                <x-button flat label="Close" x-on:click="close" />

                <x-button primary label="Confirm" wire:click="Save" />
            </x-slot>
        </x-card>
    </x-modal>
```
